edgelet/support-bundle/src/support_bundle.rs

```diff
@@ -278,89 +281,19 @@ where
             command.output()
         };
 
-        #[cfg(windows)]
-        let command = {
-            let until = until_time
-                .map(|until| format!(";EndTime='{}'", until))
-                .unwrap_or_else(String::new);
-            ShellCommand::new("powershell.exe")
-                .arg("-NoProfile")
-                .arg("-Command")
-                .arg(&format!(r"Get-WinEvent -ea SilentlyContinue -FilterHashtable @{{ProviderName='iotedged';LogName='application';StartTime='{}'{}}} |
-                                Select TimeCreated, Message |
-                                Sort-Object @{{Expression='TimeCreated';Descending=$false}} |
-                                Format-List", since_time.to_rfc3339(), until))
-                .output()
-        };
-
         let (file_name, output) = if let Ok(result) = command {
             if result.status.success() {
-                ("iotedged.txt", result.stdout)
+                (format!("{}.txt", name), result.stdout)
             } else {
-                ("iotedged_err.txt", result.stderr)
+                (format!("{}_err.txt", name), result.stderr)
             }
         } else {
             let err_message = command.err().unwrap().to_string();
             println!(
```

Comment:
I would recommend removing this println and the `err.txt` output for missing services. Since `'iotedge` is used for earlier versions, the `println!` will cause more support calls or Github issues.

Reply:
When journalctl is called for a service that doesn't exist, it simple returns empty logs with status code 0. It does not return an error. So this logic is for real errors, which users will want to see.
```bash
$ journalctl -u blah
-- Logs begin at Sun 2020-08-30 18:22:36 PDT, end at Thu 2020-10-08 15:19:28 PDT. --
-- No entries --
$ echo $?
0
```